zarr/tests/test_hierarchy.py

```diff
@@ -928,6 +928,22 @@ def create_store():
         return store, None
 
 
+try:
+    import sqlite3
+except ImportError:  # pragma: no cover
+    sqlite3 = None
+
+
+@unittest.skipIf(sqlite3 is None, 'python built without sqlite')
+class TestSQLiteStore(TestGroup):
```

Comment:
Nit pick, can we call this `TestGroupWithSQLiteStore` just for consistency with other test classes.

Reply:
Thanks for catching that. Not sure why I named differently. Updated as you suggested.